Monika After Story/game/script-stories.rpy

```diff
@@ -348,11 +412,556 @@ label mas_story_immortal_love:
     m "The husband had ordered many bouquets in advance to ensure that his beloved wife would continue to receive flowers long after his death."
     m "Speechless and stunned, the wife read the note attached to the bouquet."
     m "{i}My love for you is eternal.{/i}"
-    
     m 1dubsu "Ahh..."
     m 1eua "Wasn't that a touching story, [player]?"
     m 1hua "I thought it was really romantic."
     m 1lksdlb "But I don't want to think of either of us dying."
     m 1eua "At least the ending was very heartwarming."
     m 1hua "Thanks for listening~"
     return
+
+# Scary stories start here
+label mas_scary_story_setup:
+    show monika 1dsc
+    $ mas_temp_r_flag = mas_is_raining
+    $ scene_change = True
+    $ mas_is_raining = True
+    pause 1.0
+    $ mas_temp_m_flag = morning_flag
+    call spaceroom(start_bg="monika_gloomy_room")
+    $ morning_flag = True
+    #stop music fadeout 1.0

```

Comment:
can you fade out the music and play the ambience track multi uploaded on the discord?